### ISSUE TITLE
Support of amd format

### ISSUE THREAD

**Issue by jonlov** [kei-ito/rollup-plugin-embed-css#19]
I was trying out this plugin and I noticed it wasn't working. It is happening because I'm using amd format on rollup.

The problem is that when using amd, the style code is inserted on the `outro` and because amd format use `return` the style code is never executed. 

```javascript
define(function () { 'use strict';

var index = (function (_ref) {
    return function (_ref2) {
        var h = components.h;

        var oncreate = function oncreate() {
            // new Glide('.glide').mount()
        };

        return h(
            "div",
            { "class": "glide", oncreate: oncreate },
            h(
                "div",
                { "class": "glide__track", "data-glide-el": "track" },
                h(
                    "ul",
                    { "class": "glide__slides" },
                    h(
                        "li",
                        { "class": "glide__slide" },
                        "0"
                    ),
                    h(
                        "li",
                        { "class": "glide__slide" },
                        "1"
                    ),
                    h(
                        "li",
                        { "class": "glide__slide" },
                        "2"
                    )
                )
            )
        );
    };
});

return index;

(function (words, rules, link) {
	link.setAttribute('rel', 'stylesheet');
	link.setAttribute('href', URL.createObjectURL(
		new Blob(
			rules.map(function (rule) {
				return rule.map(function (index) {
					return words[index];
				}).join('');
			}),
			{type: 'text/css'}
		)
	));
	URL.revokeObjectURL(link.getAttribute('href'));
}(
	[".","_","src","plugins","carousel","index","css","glide","slide","{","background",":","#","000",";","}"],
	[[0,1,2,1,3,1,4,1,5,1,6,1,7,1,1,8,9,10,11,12,13,14,15]],
	document.head.appendChild(document.createElement('link'))
));

});
```

The solution would be to change `outro` to `intro` in the embed-css/index.js file so it is put before the code. However, I don't know if this breaks other formats.
https://github.com/kei-ito/rollup-plugin-embed-css/blob/e3bfa916fa93c6bce5d170d2df531ffbf2876412/src/embed-css/index.js#L41

**Comment by kei-ito** [comment]
Thank you for your report. I'll add tests for formats that rollup supports.

**Comment by kei-ito** [comment]
I published v0.1.17 that supports "format:amd". Please update this plugin and try it out.

**Comment by jonlov** [comment]
@kei-ito thanks for the quick response! Now is working 🎉
